nn_sdwan/assets/dashboards/nn_sdwan_overview.json

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
For partner-built integrations, we usually recommend adding a logo and some descriptive text to the dashboard so customers can easily understand what they're being shown.